Package.swift

```diff
@@ -6,7 +6,7 @@ import PackageDescription
 let package = Package(
     name: "honeycomb-opentelemetry-swift",
     platforms: [
-        .macOS(.v10_15),
+        .macOS(.v12),
```

Comment:
did this get bumped upstream?

Reply:
it did yeh, we tidied up the swift package version files